src/api.rs

```diff
@@ -17,14 +17,14 @@ use std::sync::Arc;
 // TODO: use the num crate?
 #[derive(Clone, Copy, Debug)]
 #[repr(C)]
-pub struct Ratio {
```

Comment:
I'm guessing you didn't mean to include this.

Reply:
yes it is a leftover.